index.js

```diff
@@ -83,28 +86,33 @@ function onChangeLanguage(event) {
     changeLanguage(languageValue);
 }
 
+function startTimer() {
+    let totalSeconds = 0;
+    let restoredTime = localStorage.getItem('totalSeconds');
+    countTimer(restoredTime || totalSeconds);
+}
+
+function countTimer(time) {
+    ++time;
+    let hours = Math.floor(time / 3600);
+    let minutes = Math.floor((time - hours * 3600) / 60);
+    let seconds = time - (hours * 3600 + minutes * 60);
+    console.log('hour', hours)
+    hours = String(hours).padStart(2, '0');
+    minutes = String(minutes).padStart(2, '0');
+    seconds = String(seconds).padStart(2, '0');
+    document.getElementById('timer').innerHTML = `${hours} : ${minutes} : ${seconds}`;
+    saveTotalSecondsValue(time);
+}
+
+function saveTotalSecondsValue(time) {
+    localStorage.setItem('totalSeconds', time);
+}
+
 return {
     saveInputValue: saveInputValue,
     onChangeLanguage: onChangeLanguage
 }
+
 })();
-console.log(store)
-
-const timerVar = setInterval(countTimer, 1000);
-let totalSeconds = 0;
-function countTimer() {
-    ++totalSeconds;
-    let hour = Math.floor(totalSeconds / 3600);
-    let minute = Math.floor((totalSeconds - hour * 3600) / 60);
-    let seconds = totalSeconds - (hour * 3600 + minute * 60);
-    if(hour < 10) {
-        hour = '0' + hour;
-    }
-    if(minute < 10) {
-        minute = '0' + minute;
-    }
-    if(seconds < 10) {
-        seconds = '0' + seconds;
-    }
-    document.getElementById('timer').innerHTML = `${hour} : ${minute} : ${seconds}`;
-}
\ No newline at end of file
+
```

Comment:
remove empty line.

You can leave 1 or 0 empty lines at the end of the file.

Reply:
Removed. Thenks.